usecases.md

```diff
@@ -160,8 +161,18 @@ impact of repeated request on the rest of the infrastructure.
 
 * As an end user, I can understand the effect of changes to policy that I am making
 
-
-
+# Platform Implementer
+
+* As a platform implementer I can make my organization's **enterprise, network and quota operators** successful.
+* As a platform implementer I can make my organization's **security administrators and compliance officers** successful.
```

Comment:
These two points seem to be a summary of the below points. Organizing it into the two groups could be very helpful for readability.

Reply:
Do you mean one group each for operators and administrators or one group each for summary and details?

Reply:
one group each for operators and administrators